pkg/vault/contracts/test/PoolFactoryMock.sol

```diff
@@ -2,21 +2,20 @@
 
 pragma solidity ^0.8.24;
 
-import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
-
 import { IBasePoolFactory } from "@balancer-labs/v3-interfaces/contracts/vault/IBasePoolFactory.sol";
-import { IHooks } from "@balancer-labs/v3-interfaces/contracts/vault/IHooks.sol";
-import { IRateProvider } from "@balancer-labs/v3-interfaces/contracts/vault/IRateProvider.sol";
 import { IVault } from "@balancer-labs/v3-interfaces/contracts/vault/IVault.sol";
-import "@balancer-labs/v3-interfaces/contracts/vault/VaultTypes.sol";
+import {
+    TokenConfig,
+    PoolRoleAccounts,
+    LiquidityManagement
+} from "@balancer-labs/v3-interfaces/contracts/vault/VaultTypes.sol";
```

Comment:
Hmm, looks like this was on purpose, hahaha.. If it was, ignore my previous comments :p , I thought we were avoiding multiple line import for VaultTypes, except in the case when it was an example for external users.

Reply:
We have gone back and forth on it. Maybe 3-4 is ok; just don't want a giant 12-line import. The benefit is you can see when you change something that requires a new import (pretty small benefit).

Though maybe we can say that for mocks they should be one-liners. I just always expand them first to make sure the straight import isn't hiding IERC20 or IRateProvider.